src/api/index.js

```diff
@@ -265,17 +281,43 @@ export function deleteCredential(id, namespace) {
   return deleteRequest(uri);
 }
 
+export function getCustomResources(group, version, type, { namespace }) {
+  const uri = getResourcesAPI(group, version, type, { namespace });
+  return get(uri).then(checkData);
+}
+
+export function getCustomResource(group, version, type, { namespace, name }) {
+  const uri = getResourcesAPI(group, version, type, { namespace, name });
+  return get(uri);
+}
+
 export async function getExtensions() {
   const uri = `${apiRoot}/v1/extensions`;
-  const extensions = await get(uri);
-  return (extensions || []).map(
+  const resourceExtensionsUri = getResourcesAPI(
+    'dashboard.tekton.dev',
+    'v1alpha1',
+    'extensions'
+  );
+  let extensions = await get(uri);
+  const resourceExtensions = await get(resourceExtensionsUri);
+  extensions = (extensions || []).map(
     ({ bundlelocation, displayname, name, url }) => ({
       displayName: displayname,
       name,
       source: getExtensionBundleURL(name, bundlelocation),
       url
     })
   );
+  return extensions.concat(
+    resourceExtensions
+      ? (resourceExtensions.items || []).map(({ spec }) => ({
+          displayName: spec.displayname,
+          name: spec.name,
+          type: spec.type,
```

Comment:
using `type` feels a little off, but `kind` could cause confusion... might revisit this before finalising and documenting it